### PR TITLE
Atlas Genpop

### PR DESCRIPTION
**Pull request by MonaraMir**
## About The Pull Request

Removes isolation cells with timers on them, replaces them with a genpop system. This PR only affects Rift(atlas).

## Why It's Good For The Game

GBJ is awful, especially when security is the one doing it to you. This will also make it easier to get event roles to play, since they wont be forced into the youtube cells to afk the rest of the round away.

Yes, there are still ways to isolate prisoners if they are attacking each other in prison, but it should hopefully be used as a last resort only.

PR #6346 must be merged before/alongside this PR

## Changelog

:cl:
add: Refurbished Genpop, and put turnstiles and prisoner lockers into effect on Atlas.
del: Deleted old timed cells on Atlas
/:cl:

### CODE REVIEW

**Comment by MonaraMir** [comment]
Ready for review, but cannot be merged until Prisoner IDs have their access fixed. Bug report is in maintainer triage.

**Comment by MonaraMir** [comment]
I suck at figuring out why my PRs wont compile when committing, but it'll compile fine on my machine. If any reviewer actually sees a problem, let me know. Else idk, shove it in anyways. Works on my machine. 👍

**Comment by MonaraMir** [comment]
Ready for testmerge once #6346 is testmerged.

File: code/game/area/Space Station 13 areas.dm
Comment (TheLordME) [inline code/game/area/Space Station 13 areas.dm:1]: you said you would move the /area/security to a seperate file

**Review by TheLordME — CHANGES_REQUESTED** [review]

File: code/game/area/Space Station 13 areas.dm
Comment (MonaraMir) [reply code/game/area/Space Station 13 areas.dm:1]: I forgor

**Review by silicons — APPROVED** [review]

**Pull request merged**